### ISSUE TITLE
0*0 image/gif requests are missing in the generated har file

### ISSUE THREAD

**Issue by Puttajunjaiah** [webmetrics/browsermob-proxy#62]
I am using browsermob proxy to capture the network traffic for further analysis but I do not get to see the full http traffic. Unfortunately, it misses out 0*0 image/gif requests. Tried whitelisting but that didn't help. 

http://sa.abc.co.uk/bbc/test/s?name=beta.xinga.page&pal_route=service_show&ml_name=barlesque&app_type=web&language=en-GB

Appreciate any quick help.

Thanks,
Putta

**Comment by xlzdew** [comment]
yep, I get the same problems, the requests are captured except some gif request, which exactly what I wanted.

**Comment by lightbody** [comment]
The link you provided is not working for me (sa.abc.co.uk is failing DNS). Can you attach the file or give me another link that works for me?
